payroll/models/hr_salary_rule.py

```diff
@@ -288,3 +281,7 @@ def _satisfy_condition(self, localdict):
                     )
                     % (self.name, self.code, repr(ex))
                 )
+
+    def _satisfy_condition_python(self, localdict):
```

Comment:
Here you do the safe_eval inside function. Which approach are we going to use? both "python" functions must behave very similar.